openforcefield/topology/molecule.py

```diff
@@ -4697,6 +4700,137 @@ def add_conformer(self, coordinates):
 
         return self._add_conformer(coordinates)
 
+    def visualize(self, backend='rdkit', width=500, height=300):
+        """
+        Render a visualization of the molecule in Jupyter
+        
+        Parameters
+        ----------
+        backend : str, optional, default='rdkit'
+            Which visualization engine to use. Choose from:
+            - rdkit
+            - openeye
+            - nglview (conformers needed)
+        width : int, optional, default=500
+            Width of the generated representation (only applicable to
+            backend=openeye)
+        height : int, optional, default=500
+            Width of the generated representation (only applicable to
+            backend=openeye)
+
+        Returns
+        -------
+        object
+            Depending on the backend chosen:
+            - rdkit, openeye -> IPython.display.Image
+            - nglview -> nglview.NGLWidget
+        """
+        from openforcefield.utils.toolkits import OPENEYE_AVAILABLE, RDKIT_AVAILABLE
+
+        backend = backend.lower()
+
+        if backend == 'nglview':
+            try:
+                import nglview as nv
+            except ImportError:
+                raise ValueError(
+                    'Attempted to visualize with NGLview but did not find it '
+                    'installed. Try conda install -c conda-forge nglview.'
+                )
+            if self.conformers:
+                trajectory_like = _OFFTrajectoryNGLView(self)
+                widget = nv.NGLWidget(trajectory_like)
+                return widget
+            else:
+                raise ValueError(
+                    'Visualizing with NGLview requires that the molecule has '
+                    'conformers.'
+                )
+        if backend == 'rdkit':
+            if RDKIT_AVAILABLE:
+                from rdkit.Chem.Draw import IPythonConsole
+                return self.to_rdkit()
+            else:
+                warnings.warn(
+                    'RDKit was requested as a visualizaiton backend but '
+                    'it was not found to be installed. Falling back to '
```

Comment:
Sorry for the very minor comment (I promise this is the last one!), but... what about rewording this as "trying to use OpenEye..."? If we say we fall back to OpenEye but then this is not available, it would be weird UX, I think.